gwpy/timeseries/core.py

```diff
@@ -1346,10 +1345,7 @@ def get(cls, channels, start, end, pad=None, scaled=None,
                 nds_kw[key] = val
 
         # try and find from frames
-        if not nds_kw and (
-                os.getenv('LIGO_DATAFIND_SERVER')
-                or os.getenv('VIRGODATA')
-        ):
+        if not nds_kw:
```

Comment:
Were `os.getenv('LIGO_DATAFIND_SERVER')` and `os.getenv('VIRGODATA')` used before, or were they vestigial and removing them has no effect on anything?